src/plugins/opensearch_dashboards_react/public/context/types.ts

```diff
@@ -48,3 +48,22 @@ export interface OpenSearchDashboardsReactContext<T extends OpenSearchDashboards
   Provider: React.FC<{ services?: T }>;
   Consumer: React.Consumer<OpenSearchDashboardsReactContextValue<T>>;
 }
+
+export type DashboardCreators = DashboardCreator[];
+
+export type DashboardCreatorFn = (event: any, history: any) => void;
+export type DashboardItemCreatorClickHandler = (
+  creatorFn: DashboardCreatorFn
+) => (event: any) => void;
+
+export interface DashboardCreator {
+  id: string; // key identifier for creator plugin/module
+  name: string; // display name for create link
+  i18nId?: string; // unique identifier for react-intl FormattedMessage
+  i18nDescription?: string; // helpful description of this i18n context
+  i18nDefaultMessage?: string; // FormattedMessage default message, in tag-replace format
+  i18nDefaultValues?: object; // FormattedMessage default value
+  i18nEntityName?: string; // translation key
+  defaultText: string; // default translation text
+  creatorFn: DashboardCreatorFn; // onClick call this
+}
```

Comment:
This isnt necessary anymore right? The type declared in common does not have the translation keys

Reply:
Yes.  Redacted.